src/node.cc

```diff
@@ -3585,6 +3593,8 @@ static void PrintHelp() {
          "  --v8-pool-size=num    set v8's thread pool size\n"
 #if HAVE_OPENSSL
          "  --tls-cipher-list=val use an alternative default TLS cipher list\n"
+         "  --ssl-[no-]def-store  use only default certificate store\n"
```

Comment:
This probably now needs 2 options to be clear.

    --use-openssl-ca         use OpenSSL's default CA store instead of bundled one
    --use-bundled-ca        use Node's bundled Mozilla CA store

I could add (default) string to one or the other based on the #define, but then the string could be a little messy. Acceptable? What do you think?